functors_task_1/student/include/impl.h

```diff
@@ -2,12 +2,25 @@
 #include "bar_serving.h"
 #include <functional>
 
+BeerBrand& operator++(BeerBrand& beer)
```

Comment:
something is wrong with declaration

Reply:
The idea was taken from:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-oper

Reply:
Sorry, my bad, you're right.
But separation is still needed, because that expression inside is not readable

Reply:
No worries.
Regarding separation: when it's pointed, it becomes obvious. Thank you